src/examples/napi_sine/napi_sine.cpp

```diff
@@ -36,9 +37,14 @@ int main(int argc, char* argv[]) {
   Random rnd(42);               // uses fixed seed for deterministic output
   std::ofstream ofs;
 
-  //  Runtime arguments:  hotgym_napi [epochs [filename]]
+  std::string encoder_params   = "{size: " + std::to_string(DIM_INPUT) + ", activeBits: 4, radius: 0.5, seed: 2019 }";
+//  std::string sp_local_params  = "{columnCount: " + std::to_string(COLS) + "}";
+  std::string sp_global_params = "{columnCount: " + std::to_string(COLS) + ", globalInhibition: true}";
+  std::string tm_params        = "{activationThreshold: 9, cellsPerColumn: " + std::to_string(CELLS) + "}";
+
+  //  Runtime arguments:  napi_sine [epochs [filename]]
   if(argc >= 2) {
-    EPOCHS = std::stoi(argv[1]);         // number of iterations (default 5000)
+    EPOCHS = std::stoi(argv[1]);         // number of iterations (default 500)
```

Comment:
500 iters seems really too little, don't think TM will learn anything. Why is this (is it?) slower? The HelloSPTM runs 5000 just fine.

Reply:
> 500 iters seems really too little

yeah, EPOCHS default is actually 5000, its the comment that is wrong.   Fixed it.

Reply:
and still, after 5000 steps the predictions are bad? (the results should be comparable, actually even identical if you make the same sine wave and fixed random as with the Algo version. )